tests/meteor-doc-fetcher.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -ex
+
+npm install -g npm@latest
+
+export PUPPETEER_DOWNLOAD_PATH=~/.npm/chromium
+
+export METEOR_LOCAL_DIR=~/.meteor
+
+export MONGO_URL="mongodb://localhost:27017/"
+
+# https://github.com/FerretDB/FerretDB/blob/main/cmd/envtool/envtool.go#L155
```

Comment:
I don't understand the purpose of this link. What does this mean? Should this be issue link instead of a link to line number which can change?

Reply:
It was supposed to show that we should keep the replica set name consistent in all places. It's mongodb-rs in envtool.go, I need to mention that properly.

Reply:
I removed the link to avoid confusion.